velox/exec/tests/PlanNodeSerdeTest.cpp

```diff
@@ -100,6 +100,35 @@ TEST_F(PlanNodeSerdeTest, crossJoin) {
   testSerde(plan);
 }
 
+TEST_F(PlanNodeSerdeTest, nestedLoopJoin) {
```

Comment:
Should we also update TEST_F(PlanNodeSerdeTest, crossJoin) ?

I expect PlanBuilder::crossJoin to be renamed to nestedLoopJoin.